### ISSUE TITLE
colspan, rowspan

### ISSUE THREAD

**Issue by michelreyes** [paulopmx/Flexigrid#103]
I use this "great" flexigrid with table friendly and it dosn't support rowspan or colspan.
what can i do?

**Comment by oor191** [comment]
I also face this problem, waiting for someone answer too

**Comment by p0tvin29** [comment]
Yes me too... has anyone wrote a quick hack for flexigrid to support colspan ?
